packages/mobile/src/navigator/Headers.tsx

```diff
@@ -30,6 +30,7 @@ export const noHeaderGestureDisabled: StackNavigationOptions = {
 export const styles = StyleSheet.create({
   headerTitle: {
     ...fontStyles.navigationHeader,
+    maxWidth: 220,
```

Comment:
Would be better to avoid hardcoding the value.

Maybe `screenWidth * 0.7`?

Reply:
yeah good idea. i went with 0.6 because the text was getting too close, think it looks good
![Screenshot_20220413-151227_Alfajores (dev)](https://user-images.githubusercontent.com/20150449/163188233-4cdd24fe-6324-4c6d-9a1b-30759c192499.jpg)